src/opening-period/form/OpeningPeriodForm.scss

```diff
@@ -56,6 +56,16 @@
   margin: 0;
 }
 
+.opening-period-text-group {
+  display: grid;
+  grid-gap: $spacing-s;
+  grid-template-columns: repeat(auto-fill, minmax(345px, 1fr));
+}
+
+.opening-period-text-group-textarea {
+  --textarea-height: 6rem;
```

Comment:
Where is this variable applied?

Reply:
It overwrites the value of the default. Probably not needed 🤔

Reply:
It is needed because the default height is too large